### PR TITLE
Created saveEndDiscussion function to update fields from a debate when closed

### LINKED ISSUE #52: Implement an update of fields when Debate is closed in the DB
Currently the fields auditors and finishTime are never set.
This fields should be set when a debate is closed

### PR DESCRIPTION
**Pull request by Naludrag**
Fixes #52 
Changed save function to return true when the save went well.
Returned always false before.

### CODE REVIEW

File: src/database/DatabaseManager.js [hunk 1]
```diff
@@ -58,13 +58,36 @@ export class DataBaseManager {
     async getAdminId(username){
         let id = null;
         logger.debug(`Getting the id of the user ${username}`);
-        await Administrator.findOne({login:username},function(err,username) {
+        let user = await Administrator.findOne({login:username},function(err,username) {
             if (err || username == null) logger.debug(`Impossible to find username`);
-            else id = username._id;
+            else{
+                logger.debug(`User found: ${username}`);
+            }
```
Comment (ghost) [inline src/database/DatabaseManager.js]: Essaye d'être constant sur la structure des blocs if comme ça :
```
if (...) 
  instruction
else 
  instruction
```
ou
```
if (...) {
  instruction
} else { 
  instruction
}
```
Mais pas un mix des deux.

File: src/database/DatabaseManager.js [hunk 2]
```diff
@@ -58,13 +58,36 @@ export class DataBaseManager {
...
+     * @returns {Promise<Query|void>} A Discussion or null if not found
+     */
+    async getDiscussion(anIdDiscussion){
+        logger.debug(`Getting the Discussions with id ${anIdDiscussion}`);
+        // Get the discussions related to the id
+        return Discussion.findOne({_id: anIdDiscussion}, function (err, discussion) {
+            if (err || discussion == null) logger.debug(`Error when requesting discussion`);
+            else {
+                console.log(discussion);
+            }
```
Comment (ghost) [inline src/database/DatabaseManager.js]: Idem pour la structure du if.

**Review by ghost — COMMENTED** [review]
I just wish to receive an answer concerning the two comments and optionally a fix commit before merging.

File: src/database/DatabaseManager.js (hunk 1 above)
Comment (Naludrag) [reply src/database/DatabaseManager.js]: Corrected in the new commit. Changed the typo for all the if structures that were incorrect in the file.

**Review by ghost — APPROVED** [review]

**Pull request merged**